directives/annotated.py

```diff
@@ -357,6 +365,22 @@ def annotate(html, section_name, annotations):
     content = u''.join(result)
     return u''.join([parts[0], parts[1], content, parts[3], parts[4]])
 
+def add_assets(app):
+    app.add_stylesheet(CSS_FILE)
+    app.add_javascript(JS_FILE)
+
+def copy_assets(app, exception):
+    if app.builder.name not in ['html', 'readthedocs'] or exception:
+        return
+    logger = logging.getLogger(__name__)
```

Comment:
Usually, we initialize the logger at the start of the module, not inside a function.